package.json

```diff
@@ -25,7 +25,7 @@
     "chalk": "2.4.1",
     "ejs": "2.6.1",
     "eslint-config-airbnb": "17.1.0",
-    "generator-jhipster": "6.5.0",
+    "generator-jhipster": "6.5.1",
```

Comment:
Should we align the `jhipster-core` too? (although it is correctly set to 6.0.4 via transitive dependency in the  .lock file).

Reply:
yes we should !